lib/minidoc/timestamps.rb

```diff
@@ -15,6 +15,28 @@ def timestamps!
     end
   end
 
+  def set(attributes)
+    if self.class.record_timestamps
```

Comment:
won't this always be true? (question applies to all the method overrides, including some not introduced in this PR)